dev-client/src/components/ScreenFormWrapper.tsx

```diff
@@ -94,7 +96,16 @@ export const ScreenFormWrapper = forwardRef(
                       size: '5',
                     }}
                     isDisabled={isSubmitting}
-                    onPress={onOpen}
+                    onPress={() => {
```

Comment:
Can you explain this change to me? I don't understand what it's doing, and the chain of dereferences in the if-statement is quite long. Maybe you could extract it to a callback / some variables in the component declaration.

Reply:
This change prevents confirm prompting on deletion on a new (previously unsaved) and empty note.

Reply:
I moved the code into a helper named `conditionallyConfirmDelete`.